docs/tutorials/add_user_project.rst

```diff
@@ -0,0 +1,60 @@
+========================================
+Documentation: Adding a User and Project
+========================================
+(Need Feedback)
+
+Adding a User
+=============
+
+OpenHatch community welcomes its users to initiate an active participation in various activities such as contributing to a project, Finding a bug and Adding your own Project.
```

Comment:
It's good style to keep the line length less than 79 characters. It helps future readers of your code to see it on a screen without the lines wrapping which helps in developer maintainability of the code. Editors may be set to remind you that the line is exceeding 79 characters.